lib/uplink/uplink.go

```diff
@@ -138,8 +138,15 @@ func NewUplink(ctx context.Context, cfg *Config) (*Uplink, error) {
 	}, nil
 }
 
+// ProjectOptions allows configuration of various project options during opening
+type ProjectOptions struct {
+	Volatile struct {
+		EncryptionKey storj.Key
```

Comment:
We usually have `storj.Key` fields and params declared as pointers to avoid having many copies of this sensitive information in the memory. Shouldn't we have it declared here as a pointer too?

Reply:
oh, i see, that's why that's happening. okay, i'll update